hello-world-master/source/components/scene/animationhelper.brs

```diff
@@ -0,0 +1,35 @@
+function translationAnimation(target as Object, originX as Integer, originY as Integer, newX as Integer, newY as Integer) as Object
+    translateAnim = createObject("roSGNode", "Animation")
+    translateAnim.duration = 1
+    translateAnim.easeFunction = "linear"
+    
+    vector2DFieldInterpolator = createObject("roSGNode", "Vector2DFieldInterpolator")
+    vector2DFieldInterpolator.key = [0.0, 1.0]
+    vector2DFieldInterpolator.keyValue = [[originX, originY], [newX, newY]]
+    vector2DFieldInterpolator.fieldToInterp = target.id+".translation"
+
+    translateAnim.appendChild(vector2DFieldInterpolator)
+    return translateAnim
+end function
+
+function parallelAnimation(animation1 as Object, animation2 as Object, animation3 as Object) as Object
```

Comment:
maybe you can get an array of animations here. In that way you can get as many animations as needed to append